src/System.Reflection/ref/project.json

```diff
@@ -1,13 +1,13 @@
 {
   "dependencies": {
-    "System.Runtime": "4.0.20",
+    "System.Runtime": "4.2.0-beta-devapi-24422-01",
     "System.IO": "4.0.0",
```

Comment:
>4.0.0 [](start = 18, length = 5)

I suppose this will also need to change to the prerelease one since you've touched the contract.